scripts/configure_dev.sh

```diff
@@ -69,13 +69,24 @@ function install_windows_shellcheck() {
 }
 
 if [ "${OS}" = "linux" ]; then
+    DISTRIB=$("$SCRIPTPATH"/distribtype.sh)
     if ! which sudo > /dev/null; then
-        apt-get update
-        apt-get -y install sudo
+        if [ "${DISTRIB}" = "ubuntu" ]; then
```

Comment:
with this change, any unspecified distribution would stop working. Just to ensure backward compatibility, could you default to ubuntu ? i.e.
```
if [ "${DISTRIB}" = "arch" ]; then
   ...
else
   # ubuntu ( or compatible) distribution
   ...
fi
```

Reply:
nice to have: Adding a warning for a non-ubuntu distributions in this case might also be a good idea.

Reply:
I have defaulted to Ubuntu in the last commit.